docs/application/dotnet/guides/nui/xaml/data-binding.md

```diff
@@ -0,0 +1,46 @@
+# Data Binding
```

Comment:
@mijin85cho, I agree with your comment and I have the same concern with most of the files. However, since the content is taken from different sites, it is important to see the copyright information and remove duplicate content wherever applicable.

Reply:
they are slightly different. the former is for using BindingContext and the latter is for using BindingSource. I want to keep this with no change anymore.